### PR TITLE
Include limbo balance in Home screen balance.

### LINKED ISSUE #1006: Add total_limbo_balance to unified balance on home screen
The `total_limbo_balance` field from pending channels should be added to the unified balance on the home screen balance sum: https://api.lightning.community/#pendingchannels

We should also add it to the `Closing` sum at the top of the channel page on desktop.

Otherwise, if the user has only one channel, and they force close it for w/e reason, their balance on the home screen will go to zero.

### PR DESCRIPTION
**Pull request by valentinewallace**
Closes #1006.

### CODE REVIEW

File: src/action/wallet.js [hunk 1]
```diff
@@ -449,6 +453,20 @@ class WalletAction {
...
+   */
+  async getLimboBalance() {
+    try {
+      const r = await this._grpc.sendCommand('PendingChannels');
+      this._store.limboBalanceSatoshis = r.totalLimboBalance;
+    } catch (err) {
+      log.error('Getting limbo balance failed', err);
+    }
+  }
+
```
Comment (tanx) [inline src/action/wallet.js]: `store.limboBalanceSatoshis` should be done in `getPendingChannels` of channel actions as that grpc api is already being polled. Doing it here would result in the api being polled twice.

File: src/computed/wallet.js [hunk 2]
```diff
@@ -21,8 +21,14 @@ const ComputedWallet = store => {
...
         channelBalanceSatoshis,
+        limboBalanceSatoshis,
       } = store;
-      return balanceSatoshis + pendingBalanceSatoshis + channelBalanceSatoshis;
+      return (
+        balanceSatoshis +
+        pendingBalanceSatoshis +
+        channelBalanceSatoshis +
+        limboBalanceSatoshis
+      );
```
Comment (tanx) [inline src/computed/wallet.js]: Although `store.limboBalanceSatoshis` is set in channel actions I would also compute this here 👍

File: test/unit/computed/wallet.spec.js [hunk 3]
```diff
@@ -51,10 +52,11 @@ describe('Computed Wallet Unit Tests', () => {
       store.balanceSatoshis = 50000001;
       store.pendingBalanceSatoshis = 50000000;
       store.channelBalanceSatoshis = 10000;
+      store.limboBalanceSatoshis = 100;
       store.settings.unit = 'bit';
       ComputedWallet(store);
-      expect(store.totalBalanceSatoshis, 'to equal', 100010001);
-      expect(store.totalBalanceLabel, 'to match', /1[,.]000[,.]100[,.]01/);
+      expect(store.totalBalanceSatoshis, 'to equal', 100010101);
+      expect(store.totalBalanceLabel, 'to match', /1[,.]000[,.]101[,.]01/);
```
Comment (tanx) [inline test/unit/computed/wallet.spec.js]: Nice test 👍

File: src/computed/wallet.js (hunk 2 above)
Comment (valentinewallace) [reply src/computed/wallet.js]: Sorry, I'm confused what this means. What is being computed?

File: src/action/wallet.js (hunk 1 above)
Comment (valentinewallace) [reply src/action/wallet.js]: fixed

File: src/computed/wallet.js (hunk 2 above)
Comment (tanx) [reply src/computed/wallet.js]: Just saying it makes sense to use the `ComputedWallet` module here. So no changes needed :)

**Pull request merged**